Modules/Tests/NetworkingTests/Remote/OrdersRemoteTests.swift

```diff
@@ -916,6 +916,56 @@ final class OrdersRemoteTests: XCTestCase {
                                                   "value": cashPaymentChangeDueAmount]]
         assertEqual(received, expected)
     }
+
+    func test_searchPOSOrders_properly_returns_parsed_orders() async throws {
+        // Given
+        let remote = OrdersRemote(network: network)
+        network.simulateResponse(requestUrlSuffix: "orders", filename: "orders-load-all")
+
+        // When
+        let result = try await remote.searchPOSOrders(siteID: sampleSiteID, searchTerm: "test", pageNumber: 1, pageSize: 25)
+
+        // Then
+        XCTAssert(result.items.count == 4)
```

Comment:
Just wondering, should we use `order-load-all` for this test? As is always going to return the 4 orders no matter the `searchTerm` we pass through. Maybe is there some other file that fits better the test? Or a new response? Not a big deal in any case since we're checking that the function parses the response.

Reply:
Mm... There's actually not much to test here since our request expects the same type of Order that we already tested in other tests. Search query doesn't produce any special results. Only the second test is meaningful to verify that we're making the request with the expected parameters.